### DISCUSSION TITLE
Please help me for music collection

### DISCUSSION THREAD

**Discussion by Yhiro910** [NAIST-SE/AttractingContributors#58] [General]
url: https://github.com/Yhiro910/AttractingContributors-niceMusic

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Reply by Yhiro910** [reply]
Thank you! I'm going to add to your repository later.

**Reply by Yhiro910** [reply]
Sorry for the delay, I sent pull request.

**Comment by 278Mt** [comment]
added! and plz help my favorite movie: https://github.com/278mt/AttractingContributors
thx;-)

**Reply by Yhiro910** [reply]
Thank you! I'm going to add to your repository later.

**Reply by Yhiro910** [reply]
Sorry for the delay, I sent pull request.

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Reply by Yhiro910** [reply]
Thank you! I sent pull request.

**Comment by sergiodl-naist** [comment]
Added one of my favorite bands and songs this past years
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by Yhiro910** [reply]
Thank you! I sent pull request.

**Comment by t-matsumoto4758** [comment]
Added!
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Reply by Yhiro910** [reply]
Thank you! I sent pull request.
